src/Environment.hpp

```diff
@@ -107,13 +108,16 @@ namespace geopm
             bool do_profile() const override;
             int timeout(void) const override;
             int debug_attach(void) const override;
-            static std::set<std::string> get_all_vars();
+            static std::set<std::string> get_all_vars(void);
             bool do_ompt(void) const override;
             std::string default_config_path(void) const override;
             std::string override_config_path(void) const override;
+            static void parse_environment_file(const std::string &settings_path,
```

Comment:
This static method could be private right?

Reply:
Nope the Environment (base class) needs access to this static public member of the EnvironmentImp class line 86 in green of this patch